zcash_primitives/CHANGELOG.md

```diff
@@ -102,7 +118,7 @@ and this library adheres to Rust's notion of
     - `Bundle`
     - `SpendDescription, SpendDescriptionV5`
     - `OutputDescription, OutputDescriptionV5`
-    - `Authorization, Authorized, MapAuth`
```

Comment:
The changes that were made to `MapAuth` in `sapling-crypto` need to be moved back into this changelog, because the `MapAuth` trait added back by this PR does not match the `MapAuth` trait in `zcash_primitives 0.13.0`. I think the only change is `&self` being changed to `&mut self`, but please check this and add the corresponding change entry.